File: M3/Practica4/Exercici3.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
n1=int(raw_input("Introduce el primer numero="))
n2=int(raw_input("Introduce el segundo numero="))
if n1<n2:
	for nt in range(n2,n1,-1):
		print nt 
elif n2<n1:
	for nt in range(n2,n1,+1):
		print nt
elif n1==n2:
	print "Los numeros son iguales", n1, ",", n2

File: M3/Practica4/Exercici3.java
import java.io.*; 
public class Exercici3 {
    public static void main(String[] args) throws IOException {
        int nx;
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
		System.out.print("Introduce el primer numero=");
        String numero1 = reader.readLine();
        int n1 = Integer.parseInt(numero1);
        System.out.print("Introduce el segundo numero=");
        String numero2 = reader.readLine();
        int n2 = Integer.parseInt(numero2);
        if(n1<n2){
			for(nx=n2;nx>n1;nx--){
				System.out.println(nx);
			}
		}
		else if(n1>n2){
			for(nx=n1;nx>n2;nx--){
				System.out.println(nx);
			}
		}
		else if(n1==n2){
			System.out.println("Los numeros son iguales="+n1+","+n2);
		}
    }
}
